### DISCUSSION TITLE
Tab changes in 3.3.0

### DISCUSSION THREAD

**Discussion by leehinde** [Sequel-Ace/Sequel-Ace#999] [Ideas] [answered]
There are three feature changes in 3.3.0 that I noticed that I wanted to ask about. I'm on Catalina, fwiw.

1. Previously the color associated with a connection was applied to the entire tab. Now, you 'just' get a dot to the right of the tab. I counted on that coloring to help me realize I was playing in the production system vs development.
2. There used to be an option to "open in a new tab" to quickly get to the same location in a new tab.
3. There was also a way to always display a tab even if only one was open. Which was useful for #2.

Might any of those features come back?

**Comment by mario-fehr** [comment]
Read https://github.com/Sequel-Ace/Sequel-Ace/pull/970 for more information. For the first and third point should be an option according to the screenshots.

**Comment by Kaspik** [comment, marked as answer]
1) was fixed - https://github.com/Sequel-Ace/Sequel-Ace/issues/1013
2) It's still there, it's just named `Duplicate connection to new tab` - `CMD`+`SHIFT`+`T`
3) It's still there, it's just native / system one now - see https://github.com/Sequel-Ace/Sequel-Ace/issues/1001#issuecomment-814965166 :)
